DrinkManagerWeb/appsettings.json

```diff
@@ -11,10 +11,8 @@
   },
   "AllowedHosts": "*",
   "AppSettings": {
-    "UserName": "admin@admin.com",
-    "UserEmail": "admin@admin.com",
     "UserPassword": "Pa$$w0rd",
```

Comment:
Tylko taki hint, w kodzie idącym do repozytorium nigdy nie podajemy w plikach prawdziwych haseł i loginów.
Możemy mieć  jakieś testowe tylko a Serwer od Continous Deployment potem może je podmieniać na te "właściwe".
To tylko taki hint, żebyście pamiętali :-)